README.md

```diff
@@ -32,6 +32,8 @@ Creates a new Oppsy object.
 - `[config]` - optional configuration object
   - `httpAgents` - the list of httpAgents to report socket information about. Can be a single http.Agent or an array of agents objects. Defaults to Http.globalAgent.
   - `httpsAgents` - the list of httpsAgents to report socket information about. Can be a single https.Agent or an array of agents. Defaults to Https.globalAgent.
+  - `dontResetNetworkCounters` - true/false. If true; do not reset network counters (request and response time) every time metrics are sent. Results in a counter-like behaviour instead of gauge.
```

Comment:
I don't like this name... either `networkMode` as a string "gauge" or "counter". Or if we want to keep it a boolean just call it `resetNetwork`.